web/client/src/library/components/search/SearchList.tsx

```diff
@@ -137,7 +137,7 @@ export default function SearchList<
         }
       }}
     >
-      <Popover className="relative flex">
+      <Popover className="relative flex w-full">
```

Comment:
i did not see any negative effects elsewhere as a result of this, but i feel like this is the change that has the most potential to do something we don't want.